### ISSUE TITLE
Update answer options for senior housing question on eligibility questionnaire

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#848]
The senior housing question on the eligibility questionnaire is currently a radio button with 4 different options. Due to the intricacies of the policy, it's actually better to understand the ages of all the people who will be living in the property, so we need to modify this slightly. Instead of a radio button, it should be checkboxes with the following options:

0 - 49 years old
50 - 54 years old
55 - 61 years old
62 years old or older
Prefer not to say

Design is viewable in [Figma](https://www.figma.com/file/wAhoXVmHRIJ2TyBKFVq2JY/Desktop-and-Mobile-Experiences?node-id=3540%3A13497) (03.3).

![eligibility_questionaire_housing_answers](https://user-images.githubusercontent.com/394908/145485124-9b3ef5c6-9775-4106-b294-2686393c2240.png)
